e2e-tests/tfm/cypress/e2e/pages/caseDealPage.js

```diff
@@ -8,6 +8,7 @@ const caseDealPage = {
   bankSecurityFacilitySubHeading: () => cy.get('[data-cy="bank-security-facility-sub-heading"]'),
   bankSecurityText: () => cy.get('[data-cy="bank-security-text"]'),
   bankSecurityFacilityText: () => cy.get('[data-cy="bank-security-facility-text"]'),
+  cancelButton: () => cy.get('[data-cy="cancel-button"]'),
```

Comment:
We have a partial for `cancelLink`, can we use that?

Not sure if the design is a button or link

Reply:
This is a button & its a different thing to the cancel link :) This button is for cancelling a deal (that is abandoning/deleting/getting rid of it), rather than cancelling a journey (aborting/ending early etc) if that makes sense? This is a big ol' red button rather than a hyperlink & as such I think it should not be commonised or use the commonised command


The naming is quite confusing, there has been some discussion about changing it (especially when it comes to cancelling during the cancel deal journey)

Reply:
Thanks @MarRobSoftwire understood, but I think we should use a partial and `clickCancelButton` command for this.

I know it's a primary/danger button, but it is still a "cancel" button.

We have a partial and`clickCancelButton` command in GEF and Portal suites, so having a different selector and click events for a single cancel button in TFM would be odd

I do agree that if its renamed to _not_ be a cancel button it should have it's own selector

Reply:
This button is semantically different to the usual `cancel` button/link - I've changed it to be called `cancelDealButton` & have data-cy `cancel-deal-button` to make it clear this is the case.

I think this is a naming at the business level that makes things confusing but regardless of this, I dont want to introduce ambiguity by referencing this as the same thing